bin/node/cli/src/command.rs

```diff
@@ -94,8 +94,8 @@ pub fn run() -> Result<()> {
 		Some(Subcommand::Base(subcommand)) => {
 			let runner = cli.create_runner(subcommand)?;
 			runner.run_subcommand(subcommand, |config| {
-				let (builder, _, _, _) = new_full_start!(config);
-				Ok(builder.to_chain_ops_parts())
+				let (client, backend, _, task_manager, .., import_queue) = new_full_up_to_import_queue!(&config);
+				Ok((client, backend, import_queue, task_manager))
```

Comment:
I guess if we make a type, the input type for the subcommands can use that type:

```
let base_components = service::base_components(&config);
Ok(base_components)
```

Reply:
(The other advantage is that we won't need to enumerate all the fields needed and where.)

Reply:
The reason you can't do that easily is because you need to create a whole bunch of stuff in order to create the `import_queue`.

Reply:
ok yes I read the conversation with Basti